app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  include ControllerHelper
+  before_action :user, only: %i[show edit update destroy]
```

Comment:
`except: :index`
wish to see a bit more meaningful method name: what is going before action? `user` is an object. that is a good name for the method in case it's being called from an action directly like:
```
def destroy
  user.destroy
end
```

speaking of your case, probably `find_user` or similar would fit better

Reply:
done